### ISSUE TITLE
Does Node js app still need to use Babel? How same V8 engine as javascript make any difference?

### ISSUE THREAD

**Issue by nellysattari** [kentcdodds/ama#439]
I assume we use Babel with our web applications since the run-time environment for Javascript is the browsers and not all browsers can support new ES features.

on the other hand, Chrome uses the same engine as Node which is V8 and Node asserts that it supports almost all Es6 features (maybe 1% still is not supported) .

Does it mean that if node is our run time for code , we don't need using Babel anymore? 
Also if we only target chrome does it same with chrome too?

Thanks

**Comment by kentcdodds** [comment]
Hi! I'm afraid I don't have time to give a super solid answer to all my open questions right now, so I just went through and answered all my AMA questions in the form of a video: https://www.youtube.com/watch?v=y23g8L8UtoQ&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u

A few things that folks can do to help make things better for others reading the questions and answers:

1. Link to when I start answering the question. Add `&t=xxmxxs` to the end of the URL like this: https://www.youtube.com/watch?v=y23g8L8UtoQ&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u&t=2m5s
2. Comment with links I referenced
3. Summarize what I talk about

Thanks!
